jersey/src/main/java/com/netflix/conductor/server/resources/GenericExceptionMapper.java

```diff
@@ -62,24 +60,22 @@ public GenericExceptionMapper(Configuration config) {
 	}
 	
 	@Override
-	public Response toResponse(Throwable t) {
-		logger.error(t.getMessage(), t);
+	public Response toResponse(Throwable exception) {
+		logger.error(exception.getMessage(), exception);
```

Comment:
`logger.error` already logs the exception message, would be helpful to log a more meaningful error message here.